extensions/kubeapi/rbac/delete.go

```diff
@@ -38,3 +38,19 @@ func DeleteGlobalRole(client *rancher.Client, globalRoleName string) error {
 	}
 	return nil
 }
+
+// DeleteClusterRoletemplate is a helper function that uses the dynamic client to delete a Custom Cluster Role template by name
+func DeleteClusterRoletemplate(client *rancher.Client, clusterRoleName string) error {
```

Comment:
To keep this function generic that can be used to delete a role template in both cluster and project contexts, can you rename DeleteClusterRoletemplate to DeleteRoleTemplate?